.storybook/theme-selectors.js

```diff
@@ -0,0 +1,16 @@
+import classic from '../src/style/themes/classic';
+import small from '../src/style/themes/small';
+import medium from '../src/style/themes/medium';
+import large from '../src/style/themes/large';
+
+/** These are for use with https://www.npmjs.com/package/storybook-addon-styled-component-theme */
+
+export const dlsThemeSelector = {
+  themes: [small, medium, large]
```

Comment:
What about None (generic) theme ?

Reply:
Good point! Now implemented, in c540b03 and 2d70a0b :thumbsup: